ui/selectors/selectors.js

```diff
@@ -280,17 +283,36 @@ export const getMetaMaskAccounts = createSelector(
   getInternalAccounts,
   getMetaMaskAccountBalances,
   getMetaMaskCachedBalances,
-  (internalAccounts, balances, cachedBalances) =>
+  getMultichainBalances,
+  getMultichainNetwork,
+  (
+    internalAccounts,
+    balances,
+    cachedBalances,
+    multichainBalances,
+    multichainNetwork,
+  ) =>
     Object.values(internalAccounts).reduce((accounts, internalAccount) => {
       // TODO: mix in the identity state here as well, consolidating this
       // selector with `accountsWithSendEtherInfoSelector`
       let account = internalAccount;
 
-      if (balances[internalAccount.address]) {
+      if (
+        isEvmAccountType(internalAccount.type) ||
+        balances[internalAccount.address]
+      ) {
         account = {
           ...account,
           ...balances[internalAccount.address],
         };
+      } else {
+        account = {
+          ...account,
+          balance:
+            multichainBalances?.[internalAccount.id]?.[
+              MultichainNativeAssets[multichainNetwork.chainId]
+            ] ?? '0x0',
```

Comment:
Do we expect hex format here? Is that a requirement from another component? Cause right now, we do not use hex in the `multichainBalances` object!

So maybe we should use `'0'` here instead?

Reply:
There are some inconsistencies, its hex in the `AccountTracker`, and decimals in the `MultichainBalance`. 

We can use '0' here and have a TODO to fix this later.